src/botPage/view/blockly/customBlockly.js

```diff
@@ -364,6 +364,38 @@ Blockly.Toolbox.TreeNode.prototype.onClick_ = function(_e) {
  * Preload all the audio files so that they play quickly when asked for.
  * @package
  */
+
+Blockly.WorkspaceAudio.prototype.load = function(filenames, name) {
+    if (!filenames.length) {
+        return;
+    }
+    try {
+        var audioTest = new window['Audio']();
+    } catch (e) {
+        return;
+    }
+    var sound;
+    for (var i = 0; i < filenames.length; i++) {
+        var filename = filenames[i];
+        if (filename.includes('delete')) {
+            filename = '/sound/delete.mp3';
+        }
+        if (filename.includes('click')) {
+            filename = '/sound/click.mp3';
+        }
+        if (filename.includes('disconnect')) {
+            filename = '/sound/disconnect.wav';
+        }
```

Comment:
Is it possible to update disconnect.wav to disconnnect.mp3?
```suggestion
   filenames.forEach(file => {
        if(file.include('delete', 'disconnect','click')){
            filename = `/sound/${file}.mp3`;
        }
        
    });
```

Reply:
This is the way that blockly developers have written the codes :D I just copied from node modules. Thanks for mentioning ..

Reply:
but please notice that filename is a full path and keep the includes seprately has less effort to write.